workflow/executor/docker/docker_windows.go

```diff
@@ -0,0 +1,25 @@
+// +build windows
+
+package docker
+
+import (
+	"compress/gzip"
+	"fmt"
+	"strconv"
+)
+
+func dockerCpCmd(containerID, sourcePath string, compressionLevel int, destPath string) string {
```

Comment:
Can we rename to `getDockerCpCmd`? And again for simplicity and to remove redundant code I would go with checking `runtime.GOOS` like above and switch two variables based on this:
- `levelFlagParam`: `-mx` on Windows and `-` on Linux (use as prefix when assigning `levelFlag`)
- `gzipCmd`: `7za.exe a -tgzip -si %s %s` on Windows and `gzip %s > %s` on Linux (use as suffix for first param in `fmt.Sprintf`)